patches/chromium/reland_set_app_container_acls_reliably_on_out_directory.patch

```diff
@@ -1,7 +1,7 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Stefan Smolen <ssmole@microsoft.com>
 Date: Tue, 10 Dec 2024 19:38:18 +0000
-Subject: [reland] Set app container ACLs reliably on out directory
+Subject: Set app container ACLs reliably on out directory
```

Comment:
I think this change is causing the current conflict with `main`, as it doesn't seem to be related to the other changes in this PR - would you mind reverting this to it's previous state?

Reply:
Done.